Adding descriptions and titles on user documents (1/3) (#10077)
gardener/gardener: docs/usage/shoot_maintenance.md

---

> description: Defining the maintenance time window, configuring automatic version updates, confine reconcilations only to happen during maintenance, adding an additional maintenance operation and list of special operations during maintenance

Reviewer: I feel that this description is too long. Could it be shortened?

Author: I think that `list of special operations during maintenance` can be removed because it does not provide actionable information to the user

Reviewer: I've added a new comment to reflect this change

> description: Defining the maintenance time window, configuring automatic version updates, confine reconcilations only to happen during maintenance, adding an additional maintenance operation and list of special operations during maintenance

Reviewer: ```suggestion
description: Defining the maintenance time window, configuring automatic version updates, confining reconciliations to only happen during maintenance, adding an additional maintenance operation, etc.
```
